# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dconf/meson-0.52.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- dconf-0.34.0/common/meson.build.vanilla	2020-02-20 22:29:19.987092719 +0000
+++ dconf-0.34.0/common/meson.build	2020-02-20 22:29:23.459092701 +0000
@@ -28,7 +28,7 @@
 
 libdconf_common_dep = declare_dependency(
   dependencies: glib_dep,
-  link_whole: libdconf_common,
+  link_with: libdconf_common,
 )
 
 libdconf_common_hidden = static_library(
